### PR TITLE
add and remove book implemented with redux

### PR DESCRIPTION
**Pull request by banlon-jones**
# Bookstore: connect to API

add book and remove book features to persist your changes in the server.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,14 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```
Comment (Meltrust) [inline src/App.js:1]: - [ ] Please, kindly update your readme to use the [Microverse template](https://github.com/microverseinc/readme-template) with all its sections (except for tests and live demo if not required).
- [ ] OPTIONAL: I suggest you remove all the text, styles, and images added by the CRA that are no longer needed.

That way, your project will be well documented and cleaner💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @banlon-jones,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- The project works as intended✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (banlon-jones) [reply src/App.js:1]: thanks for the feedback

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @banlon-jones, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**